src/unpoly/classes/request.js

```diff
@@ -366,6 +366,18 @@ up.Request = class Request extends up.Record {
     this.badResponseTime ??= u.evalOption(up.network.config.badResponseTime, this)
   }
 
+  /*-
+  Returns the underlying XMLHttpRequest instance.
+
+  @property up.Request#xhr
```

Comment:
To see the type in the docs we must extend this with a `@params` directive:

```
@property up.Request#xhr
@param {XMLHttpRequest} xhr
@stable
```

Sorry for our arcane documentation format!